tests/Settings/Controller/AuthSettingsControllerTest.php

```diff
@@ -20,7 +20,7 @@
  *
  */
 
-namespace Test\Settings\Controller;
+namespace Test\Settings\Settings\Controller;
```

Comment:
```suggestion
namespace Tests\Settings\Controller;
```

why was `Settings` added here?